core/services/vrf/v2/listener_v2.go

```diff
@@ -917,7 +917,7 @@ func (lsn *listenerV2) enqueueForceFulfillment(
 		requestID := common.BytesToHash(p.req.req.RequestID().Bytes())
 		subID := p.req.req.SubID()
 		requestTxHash := p.req.req.Raw().TxHash
-		etx, err = lsn.txm.CreateTransaction(txmgr.TxRequest{
+		etx, err = lsn.txm.CreateTransaction(context.TODO(), txmgr.TxRequest{
```

Comment:
This doesn't seem quite right. If this context is actually used we should probably pass in a non-test object.